eiffel-vocabulary/EiffelActivityCanceledEvent.md

```diff
@@ -123,28 +123,64 @@ __Format:__
 
-##### meta.security.sdm.authorIdentity
+##### meta.security.integrityProtection.signature
 __Type:__ String  
 __Format:__  
 __Required:__ Yes  
-__Description:__ The identity of the author of the event. This property is intended to enable the recipient to look up the appropriate public key for decrypting the digest and thereby verifying author identity and data integrity. The format of the author identity varies depending on the key infrastructure solution used. Note that this requires the presence of a Trusted Authority (TA) which the recipient can query for the correct public key. The identity and location of the TA must never be included in the event itself, as this would compromise the security of the solution.
+__Description:__ The signature produced by the signing algorithm.
+
+##### meta.security.integrityProtection.publicKey
+__Type:__ String  
+__Format:__  
+__Required:__ No  
+__Description:__ The producer of the event may include the relevant public key for convenience, rather than relying a separate key distribution mechanism. Note that this property, along with the rest of the event, is encompassed by the integrity protection offered via __meta.security.integrityProtection__.
+
+#### meta.security.sequenceProtection
+__Type:__ Object[]  
+__Format:__  
+__Required:__ No  
+__Description:__ An optional object for enabling verification of intact event sequences in a distributed environment, thereby protecting against data loss, race conditions and replay attacks. It allows event publishers to state the order in which they produce a certain set of events. In other words, it cannot provide any global guarantees as to event sequencing, but rather per-publisher guarantees. Every object in the array represents a named sequence of which this event forms a part. For every event including a given named sequence, the publisher SHALL increment __meta.security.sequenceProtection.position__ by 1. The first event produced in a given named sequence SHALL numbered 1.
 
-##### meta.security.sdm.encryptedDigest
+##### meta.security.sequenceProtection.sequenceName
 __Type:__ String  
 __Format:__  
 __Required:__ Yes  
-__Description:__ The encrypted digest. The cryptographic hash function and the decryption algorithm to use, similarly to the Trusted Authority (TA), must be known to the recipient. Note that the digest of the entire event is affected by the value of this property. For this reason the input to the hash function SHALL be the entire event unaltered in all parts except for this property, which SHALL be replaced by an empty string.
+__Description:__ The name of the sequence. There MUST not be two identical __meta.security.sequenceProtection.sequenceName__ values in the same event.
+
+##### meta.security.sequenceProtection.sequenceName
+__Type:__ Integer  
```

Comment:
Typo. The key should be 'position'. The same typo seems to have been copied to several other events as well.